dist/index.html

```diff
@@ -1,37 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To do list Web App</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="https://kit.fontawesome.com/4fa8c393d0.js" crossorigin="anonymous"></script>
-</head>
-<body>
 
-<div id="app-container">
+<head>
```

Comment:
Kindly fix this issue:
- When i try to run the server locally i have this error message in console:
![Screen Shot 2021-10-29 at 20 02 59](https://user-images.githubusercontent.com/22258274/139514803-2604cd43-b0b3-4e6c-bbb8-c575b9fc5f48.png)